### PR TITLE
React integration

### PR DESCRIPTION
**Pull request by hyshka**
What do you two think of this first pass?

The Python code probably isn't up to your standards so definitely give me some feedback on how I can improve it.

Basically, I used Create React App to generate the initial code in `/src/`, then wrote my own webpack config to add Django integration with `django-webpack-loader`. I set it up for now so the app should be viewable at `localhost:8000/app`.

I didn't commit any built files, and didn't add Docker yet, but for this to be viewable you need to run the webpack server `npm run start` and then open it up at the Django URL.

Django webpack loader will append links to the generated webpack bundles in the `index.html` template. By default, for development, webpack will then try to load assets from `localhost:3000`  which is why you need the webpack server running.

### CODE REVIEW

File: techmeet/apps/frontend/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# This allows npm package binaries to be available everywhere
+ENV PATH /usr/src/app/node_modules/.bin:$PATH
+
+# Set variable that we use in our webpack config to detect
+# that we're running the stack inside of Docker
+ENV VIRTUAL_ENV docker
+
+# Update, install, and cleanup
+# RUN apk update \
+    # && apk add git \
```
Comment (ashiazed) [inline techmeet/apps/frontend/Dockerfile]: Best to leave this off if it's commented out.

File: techmeet/apps/frontend/apps.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+"""Frontend Apps."""
```
Comment (ashiazed) [inline techmeet/apps/frontend/apps.py]: You can remove this file. We're not importing it in __init__.py and not defining the full app path in the settings so it's not actually being used.

**Comment by ashiazed** [comment]
:skier: This looks really great. We'll probably have some rearranging of folders still but let's do that after this is merged. @thornycrackers will check this over and then let's merge it in.

**Comment by hyshka** [comment]
I cleaned up some code based off of @ashiazed 's suggestions here.

File: techmeet/apps/frontend/Dockerfile (hunk 1 above)
Comment (hyshka) [reply techmeet/apps/frontend/Dockerfile]: Removed in https://github.com/keeners/techmeet/pull/11/commits/09ab9c85f12571b54d5ec4f790afe24f3ed1ff60

File: techmeet/apps/frontend/apps.py (hunk 2 above)
Comment (hyshka) [reply techmeet/apps/frontend/apps.py]: Removed in https://github.com/keeners/techmeet/pull/11/commits/09ab9c85f12571b54d5ec4f790afe24f3ed1ff60

**Comment by hyshka** [comment]
@thornycrackers can you review the Docker and Makefile integration for me? I'm curious what suggestions you have.

Right now for local development running both BE + FE is required.

I'd like to add CI for frontend linting, is the one `make lint_frontend` command enough to set that up with?

**Comment by thornycrackers** [comment]
@hyshka It looks good. The only thing I'd get picky over is using `bash` over `sh`, and having the docker commands in a compose file vs using `docker run` but those can be cleaned up later I'll make some tasks for them later. I can look at adding the front end linting from what you've provided.

**Pull request merged**